java/EmbeddedCampaignManagerExample.java

```diff
@@ -0,0 +1,91 @@
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.security.SecureRandom;
+import java.util.Base64;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+
+public class EmbeddedCampaignManagerExample {
+  private static final String HMAC_SHA256_ALGORITHM = "HmacSHA256";
+
+  public static void main(String[] args) {
+    String baseUrl = "https://portal.rmp.moloco.cloud";
```

Comment:
I think this should be a constant(i.e. `private static final`) so that a client only cares the variables that they should fill.

Reply:
Addressed in https://github.com/moloco-rmp-samples/sso-examples/pull/1/commits/594325f8c284a618bf5875f7be1c70a90177ab08